src/main/resources/plugin.yml

```
@@ -0,0 +1,23 @@
+main : near.vanilla.death.database.PluginLifecycle
+name : DeathDatabase
+version : 1.0

```

This could be set to `${version}` and expanded by Gradle on build time.
```suggestion
version: ${version}
```
Gradle:
```kotlin
tasks {
    processResources {
        filesMatching("plugin.yml") {
            expand("version" to rootProject.version)
        }
    }
}